src/openrct2-ui/windows/Map.cpp

```diff
@@ -1346,11 +1361,13 @@ static constexpr ScreenCoordsXY MiniMapOffsets[] = {
             if (mainViewport == nullptr)
                 return;
 
-            auto offset = MiniMapOffsets[GetCurrentRotation()];
-            auto leftTop = ScreenCoordsXY{ (mainViewport->viewPos.x >> 5) + offset.x,
-                                           (mainViewport->viewPos.y >> 4) + offset.y };
-            auto rightBottom = ScreenCoordsXY{ ((mainViewport->viewPos.x + mainViewport->view_width) >> 5) + offset.x,
-                                               ((mainViewport->viewPos.y + mainViewport->view_height) >> 4) + offset.y };
+            auto offset = MiniMapOffsetFactors[GetCurrentRotation()];
+            auto leftTop = ScreenCoordsXY{ (mainViewport->viewPos.x >> 5) + offset.x * getTechnicalMapSize() - 8,
```

Comment:
I see the `- 8` pop up in quite a few places. Maybe something for a constant?

Reply:
Yes, perhaps. What would be a good name for it? Something like `kCornerIndicatorLength`?

Reply:
On second thought, I believe they should be removed along with the ones previously removed in f7330a2.